### ISSUE TITLE
Python 3

### ISSUE THREAD

**Issue by vivald99** [freesunshine0316/neural-graph-to-seq-mp#15]
Hi @freesunshine0316 . As I said in another issue, I'm having the correct BLEU results after using the correct tokenizer. Thank you again.

I was testing the code in python 3 (with some small changes, just to run the code (e.g range/xrange)). Using python 3, I trained the model and I got very different results (unfortunately, very low bleu scores). I was wondering if you tested the code in Python 3 and could help me if some tip. 

I trained the model a few times and I did not find why the results (bleu) were different from python2. I checked the data generated by G2S_data_stream using python 2 and 3, and they are the same (with no shuffle)!

Maybe, there is some issue in some part of the model that I was not able to find yet, like [that](https://stackoverflow.com/questions/50872628/keras-model-tensorflow-backend-trains-perfectly-well-in-python-3-5-but-very-ba).

I have been looking the code several times, but I could not find the issue yet. Maybe because I'm not a tensorflow expert.

**Comment by freesunshine0316** [comment]
I the code was developed under Python 2, and I'm not very familiar with Python 3. Is it possible to just stick to Python 2?

**Comment by nguyenchiminh99** [comment]
> Hi @freesunshine0316 . As I said in another issue, I'm having the correct BLEU results after using the correct tokenizer. Thank you again.
> 
> I was testing the code in python 3 (with some small changes, just to run the code (e.g range/xrange)). Using python 3, I trained the model and I got very different results (unfortunately, very low bleu scores). I was wondering if you tested the code in Python 3 and could help me if some tip.
> 
> I trained the model a few times and I did not find why the results (bleu) were different from python2. I checked the data generated by G2S_data_stream using python 2 and 3, and they are the same (with no shuffle)!
> 
> Maybe, there is some issue in some part of the model that I was not able to find yet, like [that](https://stackoverflow.com/questions/50872628/keras-model-tensorflow-backend-trains-perfectly-well-in-python-3-5-but-very-ba).
> 
> I have been looking the code several times, but I could not find the issue yet. Maybe because I'm not a tensorflow expert.

Hi, Can you send me this sourcode in python 3? Thank you very much, I can't install tensorflow

**Comment by freesunshine0316** [comment]
Hi, 
Sorry, I'm unable to make the change. 
You can try our most recent model https://github.com/Soistesimmer/AMR-multiview, which performs better.
But you have to train by yourself.
